docs(review): docs content review week 34/2023 (#2059)
scaleway/docs-content: tutorials/deploy-umami-on-serverless-containers/index.mdx

---
meta:
  title: Deploying Umami web analytics on Serverless Containers

>   description: In this tutorial you will learn how to deploy Umami web analytics on Serverless Containers.

Reviewer: ```suggestion
  description: Discover how to deploy Umami web analytics on Serverless Containers.
```

content:
  h1: Deploying Umami web analytics on Serverless Containers

>   paragraph: In this tutorial you will learn how to deploy Umami web analytics on Serverless Containers.

Reviewer: ```suggestion
  paragraph: Discover how to deploy Umami web analytics on Serverless Containers.
```

    <Message type="important">

>       In order to run this example, you must first create a secret token from the credentials page, and replace the expression `$SCW_SECRET_TOKEN` with the created token.

Reviewer: ```suggestion
      In order to run this example, you must first create a secret token from the credentials page, and replace `$SCW_SECRET_TOKEN` with the created token.
```
